language/Greek/strings.po

```diff
@@ -2,3380 +2,3381 @@
 
 #: src/emu/inpttype.ipp:576
 msgctxt "input-name"
 msgid "P9 Pedal 2"
-msgstr ""
+msgstr "P9 Πετάλι 2"
 
 #: src/emu/inpttype.ipp:577
 msgctxt "input-name"
 msgid "P10 Pedal 2"
-msgstr ""
+msgstr "P10 Πετάλι 2"
 
 #: src/emu/inpttype.ipp:582
 msgctxt "input-name"
 msgid "P1 Pedal 3"
-msgstr ""
+msgstr "P1 Πετάλι 3"
 
 #: src/emu/inpttype.ipp:583
 msgctxt "input-name"
 msgid "P2 Pedal 3"
-msgstr ""
+msgstr "P2 Πετάλι 3"
 
 #: src/emu/inpttype.ipp:584
 msgctxt "input-name"
 msgid "P3 Pedal 3"
-msgstr ""
+msgstr "P3 Πετάλι 3"
 
 #: src/emu/inpttype.ipp:585
 msgctxt "input-name"
 msgid "P4 Pedal 3"
-msgstr ""
+msgstr "P4 Πετάλι 3"
 
 #: src/emu/inpttype.ipp:586
 msgctxt "input-name"
 msgid "P5 Pedal 3"
-msgstr ""
+msgstr "P5 Πετάλι 3"
 
 #: src/emu/inpttype.ipp:587
 msgctxt "input-name"
 msgid "P6 Pedal 3"
-msgstr ""
+msgstr "P6 Πετάλι 3"
 
 #: src/emu/inpttype.ipp:588
 msgctxt "input-name"
 msgid "P7 Pedal 3"
-msgstr ""
+msgstr "P7 Πετάλι 3"
 
 #: src/emu/inpttype.ipp:589
 msgctxt "input-name"
 msgid "P8 Pedal 3"
-msgstr ""
+msgstr "P8 Πετάλι 3"
 
 #: src/emu/inpttype.ipp:590
 msgctxt "input-name"
 msgid "P9 Pedal 3"
-msgstr ""
+msgstr "P9 Πετάλι 3"
 
 #: src/emu/inpttype.ipp:591
 msgctxt "input-name"
 msgid "P10 Pedal 3"
-msgstr ""
+msgstr "P10 Πετάλι 3"
 
 #: src/emu/inpttype.ipp:596
 msgctxt "input-name"
 msgid "Paddle"
-msgstr ""
+msgstr "Κουπί"
 
 #: src/emu/inpttype.ipp:597
 msgctxt "input-name"
 msgid "Paddle 2"
-msgstr ""
+msgstr "Κουπί 2"
 
 #: src/emu/inpttype.ipp:598
 msgctxt "input-name"
 msgid "Paddle 3"
-msgstr ""
+msgstr "Κουπί 3"
 
 #: src/emu/inpttype.ipp:599
 msgctxt "input-name"
 msgid "Paddle 4"
-msgstr ""
+msgstr "Κουπί 4"
 
 #: src/emu/inpttype.ipp:600
 msgctxt "input-name"
 msgid "Paddle 5"
-msgstr ""
+msgstr "Κουπί 5"
 
 #: src/emu/inpttype.ipp:601
 msgctxt "input-name"
 msgid "Paddle 6"
-msgstr ""
+msgstr "Κουπί 6"
 
 #: src/emu/inpttype.ipp:602
 msgctxt "input-name"
 msgid "Paddle 7"
-msgstr ""
+msgstr "Κουπί 7"
 
 #: src/emu/inpttype.ipp:603
 msgctxt "input-name"
 msgid "Paddle 8"
-msgstr ""
+msgstr "Κουπί 8"
 
 #: src/emu/inpttype.ipp:604
 msgctxt "input-name"
 msgid "Paddle 9"
-msgstr ""
+msgstr "Κουπί 9"
 
 #: src/emu/inpttype.ipp:605
 msgctxt "input-name"
 msgid "Paddle 10"
-msgstr ""
+msgstr "Κουπί 10"
 
 #: src/emu/inpttype.ipp:610
 msgctxt "input-name"
 msgid "Paddle V"
-msgstr ""
+msgstr "Κουπί V"
 
 #: src/emu/inpttype.ipp:611
 msgctxt "input-name"
 msgid "Paddle V 2"
-msgstr ""
+msgstr "Κουπί V 2"
 
 #: src/emu/inpttype.ipp:612
 msgctxt "input-name"
 msgid "Paddle V 3"
-msgstr ""
+msgstr "Κουπί V 3"
 
 #: src/emu/inpttype.ipp:613
 msgctxt "input-name"
 msgid "Paddle V 4"
-msgstr ""
+msgstr "Κουπί V 4"
 
 #: src/emu/inpttype.ipp:614
 msgctxt "input-name"
 msgid "Paddle V 5"
-msgstr ""
+msgstr "Κουπί V 5"
 
 #: src/emu/inpttype.ipp:615
 msgctxt "input-name"
 msgid "Paddle V 6"
-msgstr ""
+msgstr "Κουπί V 6"
 
 #: src/emu/inpttype.ipp:616
 msgctxt "input-name"
 msgid "Paddle V 7"
-msgstr ""
+msgstr "Κουπί V 7"
```

Comment:
I know that V means Vertical in Axis paddle but I thought that since in the Horizontal Axis doesnt mention "H" It will be a bit confusing if a simple "Κ" (Κάθετος) put in there so I followed your suggestion and put 3 letters which shows more clearly whats the meaning of the input.